packages/compartment-mapper/src/infer-exports.js

```diff
@@ -81,7 +84,9 @@ function* interpretExports(name, exports, conditions) {
     );
   }
   for (const [key, value] of entries(exports)) {
-    if (key.startsWith('./') || key === '.') {
+    // "./" is explicitly an invalid key, but that doesn't
+    // stop people from trying to use it.
+    if (key !== './' && (key.startsWith('./') || key === '.')) {
```

Comment:
I think this might be more true. `./` is also presumably an invalid name for a condition.
```suggestion
        if (key === './') {
           continue; // or no-op
        } else if (key.startsWith('./') || key === '.') {
```